src/praisonai/tests/unit/test_approval_interactive.py

```
@@ -0,0 +1,287 @@
+#!/usr/bin/env python3
+"""
+Interactive test for the human approval system.
+
+This test actually calls dangerous tools to trigger approval prompts,
+allowing you to test the human-in-the-loop functionality.
+"""
+
+import sys
+import os
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)